LICENSE

```diff
@@ -1,6 +1,7 @@
-Copyright (c) 2014 HashiCorp, Inc.
-
-Mozilla Public License, version 2.0
```

Comment:
This file needs to be skipped as well.
Instead, the header should be manually changed to:
```
Copyright (c) The OpenTofu Authors
Copyright (c) 2014 HashiCorp, Inc.

Mozilla Public License, version 2.0
```

Reply:
Will do

Reply:
Just pushed the requested changes, i.e removed the *.proto file from ignore list and manually updated the LICENSE file. We don't need to add the license file to the header ignore list as copywrite tool doesn't update the preexisting copyrights, it just adds copyrights to the files that are missing copyright entirely (verified by running copywrite headers --plan).